pkgs/development/libraries/protobuf/generic-v3.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, version, src
```

Comment:
Done for v3.x. Not applied for v2.x since src fetch methods are not identical across minor versions.
Thanks for your comment.